src/identity.rs

```diff
@@ -54,6 +54,8 @@ pub enum Error {
     Forgotten,
     #[error("BUG: identity requested {0}, but only allowed {1:?}")]
     BugInvalidIdentityRequest(Identity, Arc<WorkloadInfo>),
+    #[error("No ztunnel identity configured")]
+    NoIdentity,
```

Comment:
I think this is no longer used now that we don't error out if we can start the zt inbound